Makefile

```diff
@@ -0,0 +1,15 @@
+NODE_BIN=./node_modules/.bin
+
+requirements:
+	npm install
+
+develop: requirements
+
+test: requirements
+	$(NODE_BIN)/karma start --single-run
+
+test-dev: requirements
+	$(NODE_BIN)/karma start
+
+quality: requirements
+	$(NODE_BIN)/jshint lib/ test/
```

Comment:
A Makefile didn't seem necessary for the ui toolkit as gulp was able to do everything.  If we wrote a makefile, it would essentially wrap all the gulp tasks.  What do you think?